pom.xml

```diff
@@ -2,30 +2,80 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>ch.cern.eos</groupId>
-	<version>0.1.0</version>
-	<artifactId>hadoop-xrootd</artifactId>
-
+	<version>0.0.1-SNAPSHOT</version>
+	<artifactId>EOSConnector</artifactId>
+    <packaging>nar</packaging>
 	<repositories>
 		<repository>
 			<id>cloudera-repo-releases</id>
 			<url>https://repository.cloudera.com/artifactory/repo/</url>
 		</repository>
-	</repositories>
+    </repositories>
+    <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+    </properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.hadoop</groupId>
 			<artifactId>hadoop-common</artifactId>
-			<version>2.7.4</version>
+			<version>2.7.3</version>
 		</dependency>
 		<dependency>
 			<groupId>log4j</groupId>
 			<artifactId>log4j</artifactId>
 			<version>1.2.16</version>
-		</dependency>
-	</dependencies>
+        </dependency>
+        <dependency>
+          <groupId>org.scijava</groupId>
+          <artifactId>native-lib-loader</artifactId>
+          <version>2.2.0</version>
+      </dependency>
+      <!--
+      <dependency>
+          <groupId>ch.cern.eos</groupId>
+          <artifactId>EOSConnector</artifactId>
```

Comment:
hmm?

Reply:
This is one of the issues I hit. I think that this needs to be split internally into two packages -- the NAR part with the shared libraries and the JAR part with the java code, then you can have the jar depend on the nar and they all get installed transitively